mcodingbot/plugins/highlights.py

```diff
@@ -166,6 +183,12 @@ async def on_start(_: hikari.StartingEvent) -> None:
 async def _dm_user_highlight(
     triggering_message: hikari.Message, triggers: list[str], user_id: int
 ) -> None:
+    assert triggering_message.guild_id is not None
```

Comment:
Should we use a comment to explain why the assert is ok? Im not sure if thats something worth doing tbh.

Reply:
I don't think it's worth it